extensions/split-video-scenes/src/index.tsx

```diff
@@ -0,0 +1,289 @@
+        });
+        return;
+      }
+      const options: Alert.Options = {
+        title: "scenedetect command not found",
+        message: "The command 'scenedetect' is not installed. Do you want to install it with pipx now?",
+        primaryAction: {
+          title: "Install",
+          onAction: async () => {
+            const installCommand = 'brew install pipx && pipx install "scenedetect[opencv]" --force';
+            console.log(`Executing install command: ${installCommand}`);
+            setLogs((prevLogs) => `Executing install command: ${installCommand}\n${prevLogs}`);
+
+            if (childProcess) {
+              childProcess.kill();
+            }
+            childProcess = exec(installCommand, {
+              env: { ...process.env, HOMEBREW_NO_AUTO_UPDATE: "1" },
+            });
+
+            childProcess.stdout?.on("data", (data) => {
+              if (childProcess?.killed) {
+                return;
+              }
+              console.error(data.toString());
+              setLogs((prevLogs) => `${data.toString()}\n${prevLogs}`);
+            });
+
+            childProcess.stderr?.on("data", (data) => {
+              if (childProcess?.killed) {
+                return;
+              }
+              console.error(data.toString());
+              setLogs((prevLogs) => `${data.toString()}\n${prevLogs}`);
+            });
+
+            const exitCode = await new Promise((resolve) => childProcess!.on("exit", (code) => resolve(code || 0)));
+
+            if (exitCode === 0) {
+              setLogs("");
+              process.env.PATH = getUserShellPath();
+            } else {
+              showToast({ style: Toast.Style.Failure, title: "Error", message: "Failed to install command" });
+            }
+          },
+        },
+        dismissAction: {
+          title: "Cancel",
+        },
+      };
+      await confirmAlert(options);
+      return;
+    }
+
+    let exitCode = 0;
+
+    fs.writeFileSync(logFilePath, "");
+    const outputLogFile = fs.openSync(logFilePath, "w");
+    for (const videoPath of filePaths) {
+      const outputDir = path.join(path.dirname(videoPath), path.basename(videoPath, path.extname(videoPath)));
+
+      const command = `scenedetect -i "${videoPath}" -o "${outputDir}" ${detectionMethod} split-video; echo 'deleting log file'; rm "${logFilePath}"`;
+
+      console.log(`Executing command: ${command}`);
+      setLogs((prevLogs) => `Executing command: ${command}\n${prevLogs}`);
+
+      childProcess = spawn(command, {
+        env: { ...process.env },
+        shell: true,
+        stdio: ["ignore", outputLogFile, outputLogFile],
+        detached: true,
+      });
+      childProcess.unref();
+      attachToLogsFile();
+      exitCode = await new Promise((resolve) => childProcess!.on("close", (code) => resolve(code || 0)));
+    }
+    if (exitCode === 0) {
+      closeMainWindow();
```

Comment:
Should we popToRoot and show a toast instead(with actions to open folder) - it felt weird that the window just closed without anything

Reply:
Now it shows an HUD and the folder with the scenes in finder